xla/service/stream_pool.cc

```diff
@@ -22,8 +22,34 @@ limitations under the License.
 namespace xla {
 
 StreamPool::Ptr StreamPool::BorrowStream(se::StreamExecutor* executor) {
+  return BorrowStream(executor, stream_executor::StreamPriority::Default);
```

Comment:
Looks like the impl is already unified, so we can just unify the interface as well.